File: src/main/java/net/thedragonteam/armorplus/compat/CompatibilityJustEnoughItems.java
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.compat;

import static net.thedragonteam.armorplus.ModConfig.IntegrationsConfig.enableJEIIntegration;

/**
 * @author Sokratis Fotkatzikis - TheDragonTeam
 */
public class CompatibilityJustEnoughItems implements ICompatibility {

    @Override
    public String getMODID() {
        return "jei";
    }

    @Override
    public boolean enableCompat() {
        return enableJEIIntegration;
    }
}

File: src/main/java/net/thedragonteam/armorplus/compat/CompatibilityJustEnoughItems.kt
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.compat

import net.thedragonteam.armorplus.compat.ICompatibility

/**
 * net.thedragonteam.armorplus.compat.jei
 * ArmorPlus created by sokratis12GR on 6/21/2016 10:54 PM.
 * - TheDragonTeam
 */
class CompatibilityJustEnoughItems : ICompatibility {
    override fun loadCompatibility(phase: ICompatibility.InitializationPhase) {}

    override val modid: String
        get() = "jei"

    override fun enableCompat(): Boolean = true
}